hibernate-core/src/test/java/org/hibernate/orm/test/component/basic/ComponentTest.java

```diff
@@ -167,7 +167,7 @@ public void testComponentQueries() {
 		// The following fails on Sybase due to HHH-3510. When HHH-3510 
 		// is fixed, the check for SybaseASE15Dialect should be removed.
 		if ( ! ( getDialect() instanceof SybaseASE15Dialect ) ) {
-			s.createQuery( "from Employee e where e.person = (current_timestamp, 'steve')" ).list();
+			s.createQuery( "from Employee e where e.person = (current_timestamp, 'steve', null, null, null, null, null)" ).list();
```

Comment:
```suggestion
			s.createQuery( "from Employee e where e.person = (null, null, current_timestamp, null, 'steve', null, null)" ).list();
```

Due to property sorting you have to change this unfortunately